commands/narrator/playerinfo.js

```diff
@@ -20,8 +21,8 @@ module.exports = {
                 let cha = message.guild.channels.cache.find((channel) => channel.id === ch[b])
                 if (cha.permissionsFor(guy).has(["VIEW_CHANNEL", "READ_MESSAGE_HISTORY"])) {
                     let ro = cha.name.replace("priv-", "")
-                    let rol = ro.toLowerCase()
-                    let role = rol.replace("-", "_")
+                    let rol = ro.trim().toLowerCase()
+                    let role = rol.split("-").join("_")
```

Comment:
why replacing the replace function? you can use RegExp 
```suggestion
                    let role = rol.replace(/-/g, "_")
```